net/openvswitch/Makefile

```diff
@@ -156,12 +156,12 @@ CONFIGURE_VARS += \
 	ovs_cv_flake8=no \
 	ovs_cv_python3=no \
 	ovs_cv_sphinx=no \
+	ovs_cv_python=$(PYTHON) \
```

Comment:
why not use `$(HOST_PYTHON_BIN)` here too ?
`$(PYTHON)` var resolves to `python` ; which is the host-system's Python ; maybe it's a good idea to use the build-system's Python executable here as well;

or, is this variable used for the target's Python ?

Reply:
Build system of Open vSwitch does not differentiate host and target python, but it replaces all `@PYTHON@` in the code to be run on target.  That's why I chose to patch `run_python` to use the newly introduced `PYTHON_HOST` instead for the build process.

Reply:
ack ;
thanks for the info